crates/ott-balancer-visualization/ott-visualizationprototype-panel/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "ott-visualization",
```

Comment:
Please name this `ott-vis`. I do not want to type out this big long name every time I need to.